appcues/src/main/java/com/appcues/AppcuesViews.kt

```diff
@@ -0,0 +1,96 @@
+package com.appcues
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.widget.FrameLayout
+import androidx.compose.ui.platform.ComposeView
+
+/**
+ * AppcuesFrameView should is used when customers want to define a specific place for inflating Embed content
+ * in their app. By placing an AppcuesFrameView in their layout, then can later register this view with a frameId.
+ *
+ * appcues.registerEmbed("frame1", appcuesFrameView)
+ */
+public class AppcuesFrameView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : FrameLayout(context, attrs, defStyleAttr) {
+
+    private val composeView = ComposeView(context)
+
+    internal fun setupComposeView(): ComposeView {
+        addView(composeView)
```

Comment:
maybe come back to this - but wondering if there is ever a case where we need to check if it is already added before adding again? Thinking of the recycler view case specifically - but maybe not an issue since a new frame view would get created on the new view inflate, or if something was re-used, it would have `clearComposition` called on it prior to reuse.